django_db_logger/db_log_handler.py

```diff
@@ -27,6 +28,20 @@ def emit(self, record):
             'trace': trace
         }
 
+        if getattr(record, 'request', None) and DJANGO_DB_LOGGER_REQUEST_DETAILS:
+            kwargs['details'] = {}
+            if DJANGO_DB_LOGGER_REQUEST_DETAILS_PATH:
+                kwargs['details']['path'] = getattr(record.request, 'path', None)
+
+            if DJANGO_DB_LOGGER_REQUEST_DETAILS_METHOD:
+                kwargs['details']['method'] = getattr(record.request, 'method', None)
+
+            if DJANGO_DB_LOGGER_REQUEST_DETAILS_GET_PARAMS:
+                kwargs['details']['get_params'] = getattr(record.request, 'GET', None)
+
+            if DJANGO_DB_LOGGER_REQUEST_DETAILS_POST_PARAMS:
```

Comment:
Can we use `request.data` instead and call this `DJANGO_DB_LOGGER_REQUEST_DETAILS_DATA`: or similar?

Reply:
You mean as a combination of GET and POST?

Reply:
Unfortunately the request does not necessarily have `data` (this is a DRF feature). But I changed the behaviour as discussed: There is only one new config option `` and the new functionality in ``DatabaseLogHandler`` is nested within easily extendable methods.

Reply:
The `extract_request_details` is a much cleaner approach, but we should remove it or just leave it as an example, as this is related to the implementation.

Reply:
She sub-method was removed completely, only an example of extending ``add_details`` was kept in the README.